content/ember-data/v3/default-adapter.md

```diff
@@ -0,0 +1,10 @@
+---
+id: ember-data:default-adapter
+title: Default adapter usage
+until: '4.0.0'
+since: '3.12.0'
+---
+## deprecate store.defaultAdapter (-json-api) and the -json-api adapter fallback behavior
+A deprecation warning will be shown when the store's `defaultAdapter` is accessed.
+
+To remove this deprecation warnings, users should implement an application adapter.
```

Comment:
I think we need more details here. Specifically, we _know_ what **exact** thing to tell them to do but this message is pretty vague. We know they were using the default adapter, and we know what that adapter actually is, which means that we can tell them something like:

```suggestion
To remove this deprecation warnings, create an `app/adapters/application.js` with the following:

    import { JSONAPIAdapter } from '@ember-data/adapters';

    export default JSONAPIAdapter.extend();
```

(I am not 100% sure on the import location for this, please double check / confirm)

Reply:
Yep, that's the correct location: https://guides.emberjs.com/release/models/customizing-adapters/